packages/lesswrong/components/posts/PostsPage/PostBody.tsx

```diff
@@ -19,30 +24,57 @@ const PostBody = ({post, html, sideCommentMode}: {
     skip: !includeSideComments,
   });
   
-  const { ContentItemBody, SideCommentIcon } = Components;
+  const votingSystemName = post.votingSystem || "default";
+  const votingSystem = getVotingSystemByName(votingSystemName);
+  const voteProps = useVote(post, "Posts", votingSystem);
+  
+  const { ContentItemBody, SideCommentIcon, InlineReactSelectionWrapper } = Components;
   const nofollow = (post.user?.karma || 0) < nofollowKarmaThreshold.get();
+  const contentRef = useRef<ContentItemBody>(null);
+  let content: React.ReactNode
   
+  let highlights: Record<string,ContentReplacedSubstringComponent>|undefined = undefined;
+  if (votingSystem.getPostHighlights) {
+    highlights = votingSystem.getPostHighlights({post, voteProps});
+  }
+
   if (includeSideComments && document?.sideComments) {
     const htmlWithIDs = document.sideComments.html;
     const sideComments = sideCommentMode==="highKarma"
       ? document.sideComments.highKarmaCommentsByBlock
       : document.sideComments.commentsByBlock;
     const sideCommentsMap = mapValues(sideComments, commentIds => <SideCommentIcon post={post} commentIds={commentIds}/>)
 
-    return <ContentItemBody
+    content = <ContentItemBody
       dangerouslySetInnerHTML={{__html: htmlWithIDs}}
+      ref={contentRef}
       key={`${post._id}_${sideCommentMode}`}
       description={`post ${post._id}`}
       nofollow={nofollow}
+      replacedSubstrings={highlights}
       idInsertions={sideCommentsMap}
     />
+  } else {
+    let content = <ContentItemBody
```

Comment:
This line shadows `content` rather than reassign it, which is what caused the missing post body bug. I'm sad that eslint didn't catch this (it would have with a different config).